Modules/_ctypes/cfield.c

```diff
@@ -172,10 +179,13 @@ PyCField_FromDesc(PyObject *desc, Py_ssize_t index,
         break;
 
     case EXPAND_BITFIELD:
-        *poffset += dict->size - *pfield_size/8;
-        *psize += dict->size - *pfield_size/8;
+        if (pack)
```

Comment:
Please add curly braces.

Reply:
The current code style does not use curly braces.

Reply:
[PEP 7](https://www.python.org/dev/peps/pep-0007/#id3) says that consistency with surrounding code is a valid reason to break the rules, but also that it's an opportunity to clean up the surrounding code.

I'm ambivalent about it, because on the one hand, there's a lot of code to clean up if we really want consistency, and on the other hand, lack of curly braces feels more dangerous than something like non-standard variable naming or whitespace, since it's so easy to accidentally get a [`goto fail`](https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/) type situation.

My inclination would be to either not bother trying to be consistent or just add the curly braces on any code you add or change, though I personally wouldn't make it a blocker either way.